addons/sale/static/src/js/tours/product_configurator_tour_utils.js

```diff
@@ -6,15 +6,15 @@ function productSelector(productName) {
     return `
         table.o_sale_product_configurator_table
         tr:has(td>div[name="o_sale_product_configurator_name"]
-        h5:contains("${productName}"))
+        *:contains("${productName}"))
```

Comment:
Please avoid using a wildcard in tour utils. This could lead to indeterministic behavior on a test (e.g. the name could be repeated in `description_sale`).

Please apply this comment to other wildcards in this file.
```suggestion
        span:contains("${productName}"))
```

Reply:
Thanks, updated 👍